src/main/java/com/kryptnostic/rhizome/pods/hazelcast/BaseHazelcastInstanceConfigurationPod.java

```diff
@@ -80,7 +80,10 @@ public Config getHazelcastServerConfiguration() {
                     .ifPresent( durableExecutors ->
                             config.setDurableExecutorConfigs( durableExecutorConfigs( durableExecutors ) ) );
 
-            config.getCPSubsystemConfig().setCPMemberCount( hzConfiguration.getCPMemberCount() );
+            config.getCPSubsystemConfig().setCPMemberCount( hzConfiguration.getCpMemberCount() );
+            if(hzConfiguration.getCpMemberCount()>0) {
```

Comment:
Shouldn't this be `getCpGroupSize`?

```suggestion
            if ( hzConfiguration.getCpGroupSize() > 0 ) {
```

Reply:
Nope this is correct if cpMemberCount <= 0 it disables cp configuration so no need to set group size.